src/DetailView/DetailView.js

```diff
@@ -223,7 +223,7 @@ class DetailView extends ConnectionComponent {
 				<Header
 					title={entity.Name}
 					status={entity.Status}/>
-				<Container>
+				<Container style={{marginTop: "104px"}}>
```

Comment:
Those figures all seem just random..why the 4? And why not at least the same margin as in grid view.